src/Exception/HttpException.php

```diff
@@ -37,30 +37,19 @@ public function __construct(
         $this->code = $response->getStatusCode();
     }
 
-    /**
-     * Returns the response.
-     *
-     * @return ResponseInterface
-     */
-    public function getResponse()
+    public function getResponse(): ResponseInterface
```

Comment:
This is a BC break: https://3v4l.org/LOOiv

Is this something we want to avoid in this update to make it as smooth as possible?